apps/trustlab/src/payload/fields/colourField.js

```diff
@@ -2,15 +2,16 @@ import { deepmerge } from "@mui/utils";
 import { ColourTextField } from "@nouance/payload-better-fields-plugin/ColourText";
 import { validateHTMLColorHex } from "validate-color";
 
-const colorPicker = ({ backgroundOverrides = {}, textOverrides = {} }) => {
+const validate = (value) => validateHTMLColorHex(value) || "Invalid hex color";
+const colorField = ({ backgroundOverrides = {}, textOverrides = {} }) => {
```

Comment:
If you want to keep the color name, then may be colorSettingsField since this field adds both background and text color fields?